app/test/integration-test/src/test/resources/io/syndesis/test/itest/apiconnector/TodoApiV2ApiKeyInHeader-export/model.json

```diff
@@ -46,7 +46,7 @@
           "authenticationParameterName": "keyName",
           "authenticationParameterPlacement": "header",
           "authenticationType": "apiKey:api_key",
-          "componentName": "connector-rest-swagger-http4",
+          "componentName": "connector-rest-swagger-ttp",
```

Comment:
I think it is connector-rest-swagger-http

Reply:
Yeap, fixed.